### PR TITLE
Make build options simpler

### PR DESCRIPTION
**Pull request by Sean-StarLabs**

### CODE REVIEW

File: UefiPayloadPkg/UefiPayloadPkg.dsc [hunk 1]
```diff
@@ -315,10 +315,10 @@
   gUefiCpuPkgTokenSpaceGuid.PcdCpuDisableMtrrProgramming|TRUE
   
   # Above 4G Decode
-!if $(ENABLE_ABOVE_4G_DECODE) == TRUE
-  gUefiPayloadPkgTokenSpaceGuid.PcdEnableAbove4GDecode|TRUE
+!if $(ABOVE_4G_DECODE) == TRUE
+  gUefiPayloadPkgTokenSpaceGuid.PcdAbove4GDecode|TRUE
```
Comment (MrChromebox) [inline UefiPayloadPkg/UefiPayloadPkg.dsc]: any reason this can't be reduced to `gUefiPayloadPkgTokenSpaceGuid.PcdAbove4GDecode|$(ABOVE_4G_DECODE)` ?

Comment (Sean-StarLabs) [reply UefiPayloadPkg/UefiPayloadPkg.dsc]: That was the first attempt, and it'll throw a bunch of errors (I still have them if you want them?)

I guessed "PcdsFixedAtBuild" doesn't allow variables

Comment (Sean-StarLabs) [reply UefiPayloadPkg/UefiPayloadPkg.dsc]: Seems to work now, I guess it was something else causing it not to work!

Comment (Sean-StarLabs) [reply UefiPayloadPkg/UefiPayloadPkg.dsc]: I take it back...

**Pull request closed without merging**